Software/Python/gopigo.py

```diff
@@ -675,3 +677,52 @@ def check_version():
 			version=14
 
 	return version
+
```

Comment:
@CleoQc this is the added function to gogpigo.py and it should be tested.  This works with our Keyes remote.